src/App.css

```diff
@@ -1850,4 +1890,478 @@
+  border-radius: 12px;
+  font-weight: 500;
+}
+
+.dependency-type.blocks {
+  background: var(--priority-high);
+  color: white;
+}
+
+.dependency-type.blocked-by {
+  background: var(--priority-medium);
+  color: black;
+}
+
+.dependency-type.relates-to {
+  background: var(--priority-low);
+  color: white;
+}
+
+.dependency-content {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  font-size: 14px;
+  color: var(--text-primary);
+}
+
+.dependency-status {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  margin-right: 4px;
+}
+
+.dependency-status.completed {
+  background: var(--priority-low);
+}
+
+.dependency-status.pending {
+  background: var(--priority-medium);
+}
+
+.dependency-status.blocked {
+  background: var(--priority-high);
+}
+
+.add-dependency-button {
+  width: 100%;
+  padding: 8px;
+  background: transparent;
+  border: 2px dashed var(--border-color);
+  border-radius: 4px;
+  color: var(--text-secondary);
+  font-size: 14px;
+  cursor: pointer;
+  transition: all 0.2s ease;
+  margin-top: 8px;
+}
+
+.add-dependency-button:hover {
+  border-color: var(--primary-color);
+  color: var(--primary-color);
+}
+
+.dependency-search {
+  margin-top: 8px;
+  padding: 12px;
+  background: var(--hover-bg);
+  border-radius: 4px;
+}
+
+.dependency-search-input {
+  width: 100%;
+  padding: 8px 12px;
+  border: none;
+  border-radius: 4px;
+  background: var(--bg-primary);
+  color: var(--text-primary);
+  font-size: 14px;
+  margin-bottom: 8px;
+}
+
+.dependency-search-results {
+  max-height: 200px;
+  overflow-y: auto;
+}
+
+.dependency-search-item {
+  padding: 8px;
+  border-radius: 4px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  transition: all 0.2s ease;
+}
+
+.dependency-search-item:hover {
+  background: var(--bg-secondary);
+}
+
+.dependency-type-select {
+  width: 100%;
+  padding: 8px;
+  background: var(--bg-primary);
+  color: var(--text-primary);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  margin-bottom: 8px;
+}
+
+/* Add dependency indicator to task cards */
+.task-dependencies {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  margin-top: 4px;
+  font-size: 12px;
+  color: var(--text-secondary);
+}
+
+.dependency-icon {
+  width: 14px;
+  height: 14px;
+  opacity: 0.7;
+}
+
+.blocked-indicator {
+  background: var(--priority-high);
+  color: white;
+  padding: 2px 6px;
+  border-radius: 3px;
+  font-size: 11px;
+  font-weight: 500;
+}
+
+/* Drag and Drop Styles */
+.column.dragging {
+  background: var(--column-background);
+  box-shadow: var(--ds-shadow-raised,0 8px 16px rgba(9,30,66,0.25));
+  cursor: grabbing !important;
+}
+
+.task-list.dragging-over {
+  background: var(--hover-bg);
+  border-radius: 8px;
+  min-height: 100px;
+}
+
+.task.dragging {
+  transform: rotate(2deg) scale(1.02) !important;
+  box-shadow: var(--ds-shadow-raised,0 8px 16px rgba(9,30,66,0.25)) !important;
+  cursor: grabbing !important;
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
.task[data-dragging='true'] {
  cursor: grabbing;
  transform: rotate(2deg) scale(1.02);
  box-shadow: var(--shadow-large);
}
```
Using !important flags in CSS rules should be avoided as it breaks the natural cascading of styles and makes maintenance difficult. Consider restructuring the selectors to achieve the desired specificity.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;